src/compass/s1_geocode_metadata.py

```diff
@@ -122,6 +127,132 @@ def run(cfg, burst, fetch_from_scratch=False):
+        'gamma':burst.burst_calibration.gamma,
+        'sigma_naught':burst.burst_calibration.sigma_naught,
+    }
+
+    # define the geogrid for calbration LUT
+    calibration_radargrid = radar_grid.multilook(dec_factor,
+                                                 dec_factor)
+    calibration_geogrid = isce3.product.GeoGridParameters(
+                            geo_grid.start_x,
+                            geo_grid.start_y,
+                            geo_grid.spacing_x * dec_factor,
+                            geo_grid.spacing_y * dec_factor,
+                            geo_grid.width // dec_factor + 1,
+                            geo_grid.length // dec_factor + 1,
+                            geo_grid.epsg)
+
+    # init geocode object
+    geocode_obj = isce3.geocode.GeocodeFloat32()
+    geo_calibration.orbit = burst.orbit
+    geo_calibration.ellipsoid = ellipsoid
+    geo_calibration.doppler = isce3.core.LUT2d()
+    geo_calibration.threshold_geo2rdr = threshold
+    geo_calibration.numiter_geo2rdr = iters
+    geo_calibration.geogrid(geogrid_calibration.start_x,
+                            geogrid_calibration.start_y,
+                            geogrid_calibration.spacing_x,
+                            geogrid_calibration.spacing_y,
+                            geogrid_calibration.width,
+                            geogrid_calibration.length,
+                            geogrid_calibration.epsg)
+    calibration_group_path =\
+        f'{ROOT_PATH}/metadata/calibration_information'
+    calibration_group =\
+        geo_burst_h5.require_group(calibration_group_path)
+
+    gdal_envi_driver = gdal.GetDriverByName('ENVI')
+    for calibration_key, vec_calib in calibration_dict.items():
+        # prepare input dataset in output HDF5
+        init_geocoded_dataset(calibration_group,
+                              calibration_key,
+                              geogrid_calibration,
+                              'float32',
+                              f'geocoded {calibration_key}')
+
+        calibration_dataset =\
+            geo_burst_h5[f'{calibration_group_path}/{calibration_key}']
+
+        # prepare output raster
+        geocoded_cal_lut_raster =\
+            isce3.io.Raster(
+                f"IH5:::ID={calibration_dataset.id.id}".encode("utf-8"),
+                update=True)
+
+        # populate and prepare radargrid LUT input raster
+        lut_arr = np.zeros((radargrid_calibration.length,
+                            radargrid_calibration.width))
+        lut_gdal_raster = drv_lut_radargrid.Create(
+                        f'{scratch_path}/{calibration_key}_radargrid.rdr',
+                        radargrid_calibration.width,
+                        radargrid_calibration.length,
+                        1,
+                        gdal.GDT_Float32)
+        lut_band = outRaster.GetRasterBand(1)
+        outband.WriteArray(lut_arr)
+        outband.FlushCache()
+        outRaster = None
```

Comment:
```suggestion
        lut_band = lut_gdal_raster.GetRasterBand(1)
        lut_band.WriteArray(lut_arr)
        lut_band.FlushCache()
        lut_gdal_raster = None
```

Reply:
Marking this as resolved as it was addressed in the commit [here](https://github.com/opera-adt/COMPASS/pull/114/commits/0bf26a93329e76aa8c571c8c2e23dac8e5ceb7ea)